package.json

```diff
@@ -9,7 +9,7 @@
     "test-integration": "node test/integration",
     "start": "electron . --debug --disable-http-cache",
     "pack": "npm run build && electron-builder --dir",
-    "dist": "npm run build && electron-builder --publish=onTagOrDraft",
+    "dist": "electron-builder --publish=onTagOrDraft",
```

Comment:
`npm run build` is already made in the pipeline and no reason to do it again, since `npm run dist` is primarily a script which should be run in a ci environment.

Reply:
sounds good